pandas/core/generic.py

```diff
@@ -6172,7 +6172,11 @@ def bfill(
         AssertionError
             * If `regex` is not a ``bool`` and `to_replace` is not
               ``None``.
+
         TypeError
+            * If `to_replace` is not any of the expected types
```

Comment:
don't list the scalars, just say

scalar, array-like, dict or None

Reply:
Done!